static/src/javascripts/projects/common/modules/commercial/fiv-banner.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+import config from 'lib/config';
+import { local } from 'lib/storage';
+import { isBlocked } from 'common/modules/commercial/membership-engagement-banner-block';
+import { shouldShowReaderRevenue } from 'common/modules/commercial/contributions-utilities';
+import type { Banner } from 'common/modules/ui/bannerPicker';
+import { acquisitionsBannerFivTemplate } from 'common/modules/commercial/templates/acquisitions-banner-fiv';
+import userPrefs from 'common/modules/user-prefs';
+import {getSync as getGeoLocation} from "../../../../lib/geolocation";
+import {supportContributeURL} from "common/modules/commercial/support-utilities";
+import {addTrackingCodesToUrl, submitComponentEvent} from "common/modules/commercial/acquisitions-ophan";
+import {Message} from "common/modules/ui/message";
+import bean from "bean";
+import mediator from "../../../../lib/mediator";
```

Comment:
This resolver and `../../../../lib/geolocation` can be shortened to `lib/mediator` and `lib/geolocation`. 

Also might need to replace `"` with `'` to pass the linter. Running `make fix` can help with that.